docs/usage/self-hosted-configuration.md

```diff
@@ -11,6 +11,11 @@ Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md
 
 ## allowCustomCrateRegistries
 
+## allowGitLfs
+
+**Note** You need to also set the Git commands that Renovate passes to the `gitNoVerify` option.
```

Comment:
This isn't fully correct. Renovate isn't passing any commands to the `giNoVerify` option. Instead, I would perhaps phrase it something like
> You also need to set the `gitNoVerify` option to the appropriate values for LFS to work.
...ish.